.chloggen/lokiexporter-dots-to-underscores-label-name-normalization.yaml

```
@@ -0,0 +1,19 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: translator/loki
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Added normalization so label name to follow Prometheus label names standard
```

```suggestion
note: Normalize label names so they follow the Prometheus label names standard
```
Minor wording suggestion.